### ISSUE TITLE
Error "Unable to upload screenshot with status code: 403" when pushing changes

### ISSUE THREAD

**Issue by mdeclercq** [prismicio/slice-machine#953]
From https://github.com/prismicio/slice-machine/issues/939#issuecomment-1528790787

Users are reporting that they get an error 403 when trying to push changes. We are working on the issue.
I updated slice-machine-ui to 1.0.2 and also got an error when trying to push changes.

- `node v18.16.0`
- `npm v8.1.2`

```
    "@prismicio/client": "^6.7.3",
    "@prismicio/helpers": "^2.3.9",
    "@prismicio/next": "^1.0.3",
    "@prismicio/react": "^2.5.2",
    "@prismicio/slice-simulator-react": "^0.2.3",
    "next": "13.3.1",
    "react": "18.2.0",
    "@slicemachine/adapter-next": "^0.1.2",
    "slice-machine-ui": "^1.0.2"
```

```
An error happened while pushing your changes
Error: Unable to upload screenshot with status code: 403
    at ScreenshotsManager.uploadScreenshot (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\screenshots\ScreenshotsManager.cjs:220:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\slices\SlicesManager.cjs:394:34      
    at async Promise.all (index 0)
    at async SlicesManager.updateSliceModelScreenshotsInPlace (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\slices\SlicesManager.cjs:371:24)
    at async D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:217:44
    at async Promise.all (index 0)
    at async PrismicRepositoryManager.pushChanges (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:188:26)
    at async Module.handleRPCRequest (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\handleRPCRequest.cjs:58:11)
    at async IncomingMessage.<anonymous> (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\createRPCMiddleware.cjs:17:45)
An error happened while pushing your changes
Error: Unable to upload screenshot with status code: 403
    at ScreenshotsManager.uploadScreenshot (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\screenshots\ScreenshotsManager.cjs:220:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\slices\SlicesManager.cjs:394:34      
    at async Promise.all (index 0)
    at async SlicesManager.updateSliceModelScreenshotsInPlace (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\slices\SlicesManager.cjs:371:24)
    at async D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:217:44
    at async Promise.all (index 0)
    at async PrismicRepositoryManager.pushChanges (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:188:26)
    at async Module.handleRPCRequest (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\handleRPCRequest.cjs:58:11)
    at async IncomingMessage.<anonymous> (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\createRPCMiddleware.cjs:17:45)
An error happened while pushing your changes
Error: Unable to upload screenshot with status code: 403
    at ScreenshotsManager.uploadScreenshot (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\screenshots\ScreenshotsManager.cjs:220:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\slices\SlicesManager.cjs:394:34      
    at async Promise.all (index 0)
    at async SlicesManager.updateSliceModelScreenshotsInPlace (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\slices\SlicesManager.cjs:371:24)
    at async D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:217:44
    at async Promise.all (index 0)
    at async PrismicRepositoryManager.pushChanges (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:188:26)
    at async Module.handleRPCRequest (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\handleRPCRequest.cjs:58:11)
    at async IncomingMessage.<anonymous> (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\createRPCMiddleware.cjs:17:45)
An error happened while pushing your changes
Error: Unable to upload screenshot with status code: 403
    at ScreenshotsManager.uploadScreenshot (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\screenshots\ScreenshotsManager.cjs:220:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\slices\SlicesManager.cjs:394:34      
    at async Promise.all (index 0)
    at async SlicesManager.updateSliceModelScreenshotsInPlace (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\slices\SlicesManager.cjs:371:24)
    at async D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:217:44
    at async Promise.all (index 0)
    at async PrismicRepositoryManager.pushChanges (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:188:26)
    at async Module.handleRPCRequest (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\handleRPCRequest.cjs:58:11)
    at async IncomingMessage.<anonymous> (D:\GitHub\prismic-next-startchine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:217:44
    at async Promise.all (index 0)
    at async PrismicRepositoryManager.pushChanges (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\managers\prismicRepository\PrismicRepositoryManager.cjs:188:26)    at async Module.handleRPCRequest (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\handleRPCRequest.cjs:58:11)
    at async IncomingMessage.<anonymous> (D:\GitHub\prismic-next-starter\node_modules\@slicemachine\manager\dist\_node_modules\r19\dist\createRPCMiddleware.cjs:17:45)
```

I tried deleting cookies and locale storage.
After that everything worked. But the next time I tried to push, I got this error again and could no longer fix it.

There was no such problem with the slice-machine-ui 1.0.0 and 1.0.1

_Originally posted by @vagmen in https://github.com/prismicio/slice-machine/issues/939#issuecomment-1528790787_

**Comment by mdeclercq** [comment]
For anyone having the issue, does deleting the `~/.prismic` file help?

**Comment by mdeclercq** [comment]
This issue happens when sessions are running for a long time (more than 2 hours). 

Until the fix is deployed, the workaround is restarting SliceMachine.

**Comment by mdeclercq** [comment]
Closing this, it is fixed in the version [1.0.3](https://github.com/prismicio/slice-machine/releases/tag/slice-machine-ui%401.0.3).
